rust/src/oid4vp/holder.rs

```diff
@@ -459,11 +459,19 @@ pub(crate) mod tests {
         }
 
         async fn verification_method(&self) -> String {
-            DidMethod::Key.vm_from_jwk(&self.jwk()).await.unwrap()
+            DidMethod::Key
+                .vm_from_jwk(&self.jwk())
+                .await
+                .unwrap()
+                .id
+                .to_string()
         }
 
         fn did(&self) -> String {
-            DidMethod::Key.did_from_jwk(&self.jwk()).unwrap()
+            DidMethod::Key
+                .did_from_jwk(&self.jwk())
+                .unwrap()
+                .to_string()
```

Comment:
I realize we were calling `.unwrap()` here. We should probably handle this error more gracefully.

Reply:
Tiago: Might need an alternative version that takes an actual `JWK` object instead of the `&str`. Also can improve safety using Results and strict types here. Save for a later PR (out of scope for this PR).